### ISSUE TITLE
Bloated native-base-theme folder

### ISSUE THREAD

**Issue by samratshekhar** [GeekyAnts/NativeBase#2094]
## node, npm, react-native, react and native-base version
- node `v8.9.4`
- npm `v5.7.1`
- react-native `v0.53.0`
- native-base `v2.5.1`
## Expected behaviour
Theme folder ideally should contain the overrides for components being used and should be updated if need arise
## Actual behaviour
Contains overrides for every component, which in turn bloats the project
## Steps to reproduce should include code snippet and screenshot
`node node_modules/native-base/ejectTheme.js` mentioned in http://docs.nativebase.io/Customize.html#theaming-nb-headref creates a new folder `native-base-theme` at the root of the project.
## Is the bug present in both iOS and Android or in any one of them?
Cross-platform
## Any other additional info which would help us debug the issue quicker.
Ideally `ejectTheme.js` should additionally/optionally account for a list of component for which the theme files are actually needed and generate them accordingly, instead of randomly dumping all the files, which might never be used, simply increasing the size and decreasing the maintainability of the project.

**Comment by SupriyaKalghatgi** [comment]
> Ideally ejectTheme.js should additionally/optionally account for a list of component for which the theme files are actually needed and generate them accordingly,

Can you brief on this

**Comment by samratshekhar** [comment]
Say my use case is limited to the usage of Header, BottomTabBar and Radio. In such a case what I would ideally want is to run something like `node node_modules/native-base/ejectTheme.js --components [Header, BottomTabBar, Radio]` and the resulting folder will contain just these components under `native-base-theme\components`

**Comment by SupriyaKalghatgi** [comment]
No, this is something not good use case
https://github.com/GeekyAnts/NativeBase/issues/830#issuecomment-394171422 this use case is suggested instead

**Comment by samratshekhar** [comment]
Thanks for providing the additional info. That was helpful. As for my situation, I moved `variables/platform.js` to a separate directory and deleted the `native-base-theme` folder. Next, as mentioned in https://github.com/GeekyAnts/NativeBase/issues/830#issuecomment-302386493, I imported `getTheme` from `native-base/dist/src/theme/components` and all my custom theming which was limited to color and size worked as expected.
